Meta/Lagom/Tools/CodeGenerators/LibWeb/GenerateCSSPropertyID.cpp

```diff
@@ -195,6 +196,14 @@ PropertyID property_id_from_string(StringView string)
     });
 
     generator.append(R"~~~(
+    if (string.starts_with("--"sv) && string != "--" && !is_ascii_digit(string.to_string()[2])) {
+        auto is_forbidden_character = is_any_of("\0\t\r\n !\"#$%&\'()*+,./:;<=>?[]\\^´`\u0040"sv);
```

Comment:
I don't think this forbidden character set is complete.
Looking at the spec at https://drafts.csswg.org/css-syntax-3/#typedef-ident-token
(Admittedly I'm not 100% certain this is where we have to be looking)
And scrolling down to [<ident-token> diagram](https://drafts.csswg.org/css-syntax-3/#ident-token-diagram) it is phrased quite differently.

So maybe it is easier to match the wording of that?
Of course we can later look at potential optimizations.

Also I don't even know if the rest of the engine does this correctly perhaps there is some is_valid_custom_property_identifier method